BAlleleFreqFormat.py

```diff
@@ -0,0 +1,131 @@
+from vcf.parser import _Format
+from IlluminaBeadArrayFiles import RefStrand
+import numpy as np
+
+REVERSE_COMPLIMENT = {"A": "T", "T": "A", "G": "C", "C": "G", "I": "I", "D": "D"}
+
+
+def extract_alleles_from_snp_string(snp_string):
+    """
+    Splits SNP string into tuple of individual alleles
+
+    Args: snp_string (string): allele string of the format
+        [T/C] or [G/C]
+
+    Returns:
+         allele1 (string), allele2 (string)
+    """
+    (allele1, allele2) = snp_string[1:4].split("/")
+    assert allele1 in "ATGCID", "allele %r is invalid (expected A,T,G,C,I,D)" % allele1
+    assert allele2 in "ATGCID", "allele %r is invalid (expected A,T,G,C,I,D)" % allele2
+    return allele1, allele2
+
+
+def normalize_alleles_by_strand(snp_string):
```

Comment:
Name here is misleading, should probably just indicate that it extract the reverse complement